keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -490,17 +738,19 @@ deviceMixins:
       cudaComputeCapability:
         version: 8.0.0
 - name: common-gpu-capacities
-  composite:
-    capacity:
-      memory:
-        quantity: 40Gi
+  capacity:
+    memory: 40Gi
+  sharedCapacityConsumed:
+  - name: memory
+    capacity: 40Gi
```

Comment:
I'm not sure I understand this:
```
- name: common-gpu-capacities
  capacity:
    memory: 40Gi
  sharedCapacityConsumed:
  - name: memory
    capacity: 40Gi
```

What is the distinction between the `capacity` of 40GB of memory and the `sharedCapacityConsumed` of 40GB of memory? Is this to support the case where some capacity might be local to the mixin, but other capacity comes from elsewhere? What does it even mean for capacity to be local to a mixin?

Reply:
The idea is to separate the capacity that is consumed from the shared pool from what is advertised as capacity available to workloads. So in these scenarios, we would expect the `sharedCapacityConsumed` to be larger than the `capacity`. And they obviously don't necessarily have to have matching names.